docs/linters.md

```diff
@@ -10,3 +10,14 @@ There is currently no way to ignore files outside of the `.gitignore` file.
 
 There is currently a bash issue where the exit code is not getting detected when there is multiple commands being ran.
 The easy solution for now is to return the value 1 from the function indicating an error has occurred and terminating the `make test/lint` command that triggered it
+
+## Migrations Linter ##
```

Comment:
Ah, I more meant adding documentation on the linter framework that was added. The migrations linter also makes sense (for Outreach internal docs), but here I think we should briefly document the per-project linters!